package.json

```diff
@@ -22,7 +22,7 @@
     "lerna-changelog": "^1.0.1",
     "np": "^6.0.2",
     "prettier": "2.7.1",
-    "rimraf": "^2.6.3",
+    "rimraf": "^4.2.0",
```

Comment:
Hey @mfranzke 👋 

Can you do me a favor and run
```
npm install rimraf@4.2.0
```
from the root of this project? CI is currently failing, as this dependency update is not reflected in `./package-lock.json` (causing `npm ci` to fail)

Reply:
Hey @rwaskiewicz, yes, sure, I sadly missed that.